dev-manuals/services/minigames/configuring-chickenshock.md

```diff
@@ -0,0 +1,8 @@
+# Configuring the Chickenshock minigame
+
+Game specific properties can be configured in the file `Assets/Scripts/Properties/Chickenshock.properties` in the `chickenshock` repository.  
+The file follows the format of [Java properties](https://docs.oracle.com/en/java/javase/17/docs/api//java.base/java/util/Properties.html).  
+Currently, the following things can be configured:
+  - `ingame.playtime` (in seconds, default `50`): Duration for which the user can play the game
+  - `REST.saveRound` (default `api/v1/minigames/chickenshock/results`): backend path where to save data of one round
+  - `REST.getQuestions` (default `api/v1/minigames/chickenshock/configurations/{id}/questions`): backend path for retrieving all questions for a run
```

Comment:
```suggestion
  - `REST.saveRound` (default `minigames/chickenshock/api/v1/results`): backend path where to save data of one round
  - `REST.getQuestions` (default `minigames/chickenshock/api/v1/configurations/{id}/questions`): backend path for retrieving all questions for a run
```

The api paths are outdated. This is not a problem of this pr but has to be changed though

Reply:
Before we forget it afterwards, it should be changed now.

Reply:
How sure are you that your suggestion is the current state?
Because if you're sure that your suggestion is accurate, I'm going to apply it and merge the PR.